spec/Section 3 -- Type System.md

```diff
@@ -818,6 +818,9 @@ of rules must be adhered to by every Object type in a GraphQL schema.
          characters {"__"} (two underscores).
       2. The argument must accept a type where {IsInputType(argumentType)}
          returns {true}.
+      3. If the field is a Oneof Field:
+         1. The field must be nullable.
```

Comment:
I think you mean "The argument" here and on the next line.

Reply:
Good catch :raised_hands: